yogabprawira/prime.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import "fmt"
+
+func isPrime(n int) bool {
+	if n <= 1 {
+		return false
+	}
+	for i := 2; i < n; i++ {
+		if n%i == 0 {
+			return false
+		}
+	}
+	return true
+}
+
+func main() {
+	start := 3
+	i := start
```

Comment:
kenapa initialize `start` klo cmn d pake sekali dan itu pun cmn pass ke `i` value nya?

Reply:
got it